pywbemtools/pywbemcli/_cmd_profile.py

```diff
@@ -186,7 +186,7 @@ def cmd_profile_list(context, options):
     """
     output_format = validate_output_format(context.output_format, 'TABLE')
 
-    server = context.wbem_server
+    server = context.pywbem_server.wbem_server
```

Comment:
I suggest to clean up the name here as well for consistency and rename `server`to `wbem_server`.

Reply:
I agree.  Cleaned up.